wallet/commands.go

```diff
@@ -683,6 +683,13 @@ func (wal *Wallet) GetAllProposals() {
 	}()
 }
 
+func (wal *Wallet) ClearProposals() {
```

Comment:
This function isn’t needed, it’s accessible directly from the multi wallet variable.